gordo_components/serializer/pipeline_into_definition.py

```diff
@@ -84,34 +85,27 @@ def _decompose_node(step: object, prune_default_params: bool = False):
     """
 
     import_str = f"{step.__module__}.{step.__class__.__name__}"
-    init_params = inspect.getfullargspec(step.__class__.__init__).args
+    params = step.get_params(deep=False)
```

Comment:
Should this not be guarded by a hasattr?

Reply:
Or do we not support non-scikit-learn stuff (although that would be a bit lame)

Reply:
ya.. I saw this and thought too. But I mean, I don't know _how_ you could fit a step of a pipeline, into a pipeline, without it being a BaseEstimator. And by that time, seems like a bag of worms?

Reply:
I think it is ok to assume BaseEstimator...